app/controllers/application_controller.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class ApplicationController < ActionController::Base
+  include ApplicationHelper
+  def signout
+    @current_user = nil
+    session[:user_id] = nil
+    cookies.permanent[:remember_token] = nil
```

Comment:
In your current implementation this is already `nil` anyway, either way once you fix that you should really use the `delete` method: `cookies.delete(:remember_token)`, and if you're storing `user_id` cookies you should also do `cookies.delete(:user_id)`.

[**REQUIRED**]